### ISSUE TITLE
Using animation tool twice bug

### ISSUE THREAD

**Issue by RichardB9** [nens/threedi-results-analysis#460]
Using the animation tool to load and display results the first time goes okay, but when you create a new project and try to load some results again the animation tool fails on a runtime error:

```
RuntimeError: wrapped C/C++ object of type QgsVectorLayer has been deleted 
Traceback (most recent call last):
  File "/home/nens/.local/share/QGIS/QGIS3/profiles/default/python/plugins/ThreeDiToolbox/tool_animation/map_animator.py", line 355, in update_results
    provider = layer.dataProvider()
RuntimeError: wrapped C/C++ object of type QgsVectorLayer has been deleted
```

Steps to reproduce the bug:
1. Load some results (netcdf) with the tool result selector.
2. Activate the animation tool and display some data by moving the time-slider
3. Create a new qgis-project (but dont quit Qgis).
4. Repeat step 1 and 2.

**Comment by Jonasvsl** [comment]
Can you give a time estimate for fixing this?

**Comment by leendertvanwolfswinkel** [comment]
fixed in 1.20
